src/index.ts

```diff
@@ -120,6 +127,15 @@ export function createSelectorCreator<
       ...finalMemoizeOptions
     )
 
+    // @ts-ignore
+    const makeAnObject: (...args: unknown[]) => object = memoize(
```

Comment:
:hand: Should this be moved inside the `NODE_ENV` check?

Reply:
it could, but i think that would be worse for performance? this creates one function when the selector is created, moving it inside would make a new function every time the check runs